lib/api/src/store.rs

```diff
@@ -41,6 +41,19 @@ impl Store {
         *m = handler;
     }
 
+    /// Catches any Wasm traps that happen within the execution of `closure`,
+    /// returning them as a `Result`.
+    ///
+    /// # Safety
+    ///
+    /// Highly unsafe since `closure` won't have any destructors run.
```

Comment:
Won't have any destructors run on the closure itself or the things contained inside it?